fuse_examples/segmentation/siim/runner_seg.py

```diff
@@ -105,7 +89,7 @@
          'force_reset_model_dir': True,  # If True will reset model dir automatically - otherwise will prompt 'are you sure' message.
```

Comment:
Let's not assume anything about ROOT and DATA_ROOT.
Please document what each one is, and ask the user to fill it in